src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,279 @@
+                    <slot
+                      name="column-1-item-0"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-1-item-1"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-1-item-2"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                  <li class="o-footer__main-content-panel-list-item">
+                    <slot
+                      name="column-1-item-3"
+                      @click="${this._handleLinkClick}"
+                    />
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <div class="o-footer__social-media">
+              <slot
+                name="column-2-social-title"
+                class="o-footer__social-media-title"
+              ></slot>
+              <ul class="o-footer__social-media-list">
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-0"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-1"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-2"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-3"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-4"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+                <li class="o-footer__social-media-item">
+                  <slot
+                    name="column-social-item-5"
+                    @click="${this._handleLinkClick}"
+                  ></slot>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </axa-container>
+      </footer>
+    `;
+  }
+
+  updated() {
+    this._removeEmptyListElements();
+  }
+
+  _removeEmptyListElements() {
+    this.shadowRoot
+      .querySelectorAll('.o-footer__main-content-panel-list-item')
```

Comment:
never use style classes for JS selection. If u want to change style or class name on style u need to refactor as well JS. Please add another class named like so: `.js-footer__main-content-panel-list-item` for example. We always prefix with `js` for dom modifications

Reply:
Good point!